self-host/entrypoint.sh

```diff
@@ -154,6 +154,9 @@ if [ "$api_key" == "null" ] || [ "$api_secret" == "null" ] || [ "$management_key
   mv "$mounted_temp" "$mounted_config"
 fi
 
+# Magic Links
+export MagicLinks__NewAccountTimeout="0.00:00:00"
+
 # Configure overrides for the admin console app
 jq '.ApplicationOverrides.admin.IsRateLimitBypassEnabled = true | .ApplicationOverrides.admin.IsMagicLinkQuotaBypassEnabled = true' \
```

Comment:
Do we know why this line isn't enabling the rate limiting and magic link quota bypass?  This seems like it was supposed to solve this problem.